library/ssl_tls13_client.c

```diff
@@ -2739,6 +2739,11 @@ static int ssl_tls13_postprocess_new_session_ticket( mbedtls_ssl_context *ssl,
                            session->resumption_key,
                            session->resumption_key_len );
 
+    /* Set ticket_flags according to the key exchange mode */
+    mbedtls_ssl_session_set_ticket_flags_psk( session,
+                                              ssl->handshake->key_exchange_mode );
```

Comment:
```suggestion
                                              ssl->handshake->tls13_kex_modes );
```

Reply:
`tls13_kex_modes` belongs to SRV code, we don't have it on client side. I could move it to be shared by both server and client.